examples/mongodbatlas_stream_processor/README.md

```diff
@@ -0,0 +1,11 @@
+# MongoDB Atlas Provider - Atlas Stream Processor defined in a Project
+
+This example shows how to use Atlas Stream Processors in Terraform. It also creates a project, which is a prerequisite.
+
+You must set the following variables:
+
+- `public_key`: Atlas public key
+- `private_key`: Atlas  private key
+- `project_id`: Unique 24-hexadecimal digit string that identifies the project where the stream instance will be created.
```

Comment:
I think `pipeline` should be an argument and also `state`. Can add some descriptions and show how to use `jsonencode()` instead of a raw string. Also, link to more documentation

Reply:
> I think pipeline should be an argument and also state. 
I don't think it makes much of a difference to include those attributes as arguments for the example if documentation is clear. 

>  Can add some descriptions and show how to use jsonencode() instead of a raw string. Also, link to more documentation
I will do this in the schema/docs description so that it's clear what this field expects and how jsonencode() should be used. Will also change the example so it uses the jsonencode()
 WDYT?

Reply:
SGTM

Reply:
Done!